src/libraries/System.Threading.Channels/src/System/Threading/Channels/ChannelOptions.cs

```diff
@@ -56,7 +56,7 @@ public BoundedChannelOptions(int capacity)
                 throw new ArgumentOutOfRangeException(nameof(capacity));
             }
 
-            Capacity = capacity;
+            _capacity = capacity;
         }
```

Comment:
Why we do not remove repeated code?
I think we can use as bellow:

`public BoundedChannelOptions(int capacity)
        {
            Capacity = capacity;
        }`